structure/0006-governance.md

```diff
@@ -2,53 +2,135 @@
+If it is decided that more administrative board members are required, either due to a current member stepping down, or due to it being decided that the admin board needs to be expanded, an election must take place.
 
-#### Explanation
-What will this team *do*? What new projects will be created that this team will be responsible for and how will each of these projects benefit the QuiltMC organization/community? What processes will this team follow for each of the different kinds of changes their projects may undergo?
+### Admin board Voting Process
 
-#### Drawbacks
-Can usually be omitted. Is only necessary if there are clear drawbacks to such a teams existence. Drawbacks may include overlap of responsibilities, excessive granularity, etc.
+ Due to the fact that multiple candidates may be nominated for the admin position when a vote is held, the fact that there may be multiple spots to fill on the board, and the fact that the decision affects the entire organization, the standard voting procedure cannot be used. Instead, the following procedure will be employed:
 
-#### Rationale and Alternatives
-Can be omitted. Who is currently responsible for existing projects this team will takeover? What teams might become responsible for new projects proposed in this document if the team itself is deemed unnecessary?
+Before the vote begins, a group of people, henceforth referred to as the Election Oversight Board or EOB, must be selected to oversee the voting process. They will be responsible for receiving candidate nominations, concerns about nominated candidates from other organization members, and hosting the election. In normal circumstances, the EOB consists of the existing members of the admin board, but, if this is not possible for any reason, the EOB will instead consist of members with the Community Manager role in the Community Team. The initial EOB may add any members that they feel would be a valuable addition to the board, with a simple majority vote.
 
-#### Prior Art
-Can usually be omitted. Is only necessary if there are prior organizations/projects with very similar goals.
+The vote is conducted using [Single Transferrable Vote](https://en.wikipedia.org/wiki/Single_transferable_vote) (STV), or [Instant Runoff voting](https://en.wikipedia.org/wiki/Instant-runoff_voting) (IRV) in instances where there is only one space on the admin board to be filled, to ensure a minimum amount of "wasted votes". When using Single Transferrable vote, the [Droop Quota](https://en.wikipedia.org/wiki/Droop_quota) must be used to calculate the amount of votes required for a candidate to win a position on the admin board.
+
+At the start of the voting process, each remaining board member may nominate an organization member, and organization members may also nominate themselves. Nominations must be submitted to the EOB within one week of the start of the process, after which time the vote is opened.
```

Comment:
We haven't seen any nomination of someone else in practice. This would honestly easily make the person being nominated uncomfortable if they don't want to run for the election, or can be seen as an endorsement otherwise which can spark a conflict of interest. I'd suggest to remove that. 
```suggestion
At the start of the voting process, each organization members may nominate themselves. Nominations must be submitted to the EOB within one week of the start of the process, after which time the vote is opened.
```